AdminPortal/src/pages/dashboard.tsx

```diff
@@ -1,6 +1,284 @@
-/**
- * Renders the {url}/dashboard page.
- */
-export default function Dashboard() {
-  return <div>{'This is the dashboard!'}</div>;
+import React, { useState, useEffect } from 'react';
+import styles from '@/styles/Dashboard.module.css';
+import Dropdown from 'react-bootstrap/Dropdown';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import data from '../data/data.json';
+import { Lato } from '@next/font/google';
```

Comment:
We shouldn't have to do this because the font is imported in globals.css

Reply:
true, have removed it